test/internal/modules/charge-information/controllers/charge-element.js

```diff
@@ -100,8 +100,8 @@ const validPayload = {
     'endDate-month': '10'
   },
   quantities: {
-    authorisedAnnualQuantity: 1234,
-    billableAnnualQuantity: null
+    authorisedAnnualQuantity: '1234',
+    billableAnnualQuantity: ''
```

Comment:
No drama but I think you have since made this nullable so feel free to swap out `''` for `null` (Or don't - It doesn't matter)

Or better still: Feel free to have tests for both null and `''`

Reply:
I changed this from a number to a string so that I could validate it with a regex, so I don't think that it will ever be a null